### DISCUSSION TITLE
原创文章 - deepbook 详细解读

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [starcoinorg/web3startrek#116] [Call for Article]
Move 生态技术文章

任务描述：撰写原创文章+直播讲解文章

题目：deepbook 详细解读

要求示例：
1. 为什么要 deepbook；
2. 如何使用 deepbook
3. deepbook 源码解读；
4. 如何影响 DeFi 生态。

截止时间：2023.04.25

任务奖励：70U + MoveFuns DAO 100积分    23号以前完成 + 20SUi

首发：DAOrayaki 策展

任务跟进： @geometryolife @uvd

**Comment by zgfzgf** [comment]
我可以试一试，时间有点赶

**Reply by ShirleneLiu** [reply]
加一下我vx吧：shirleneliu420
